package-lock.json

```diff
@@ -4765,6 +4799,7 @@
     "eslint-plugin-zillow": {
       "version": "file:packages/eslint-plugin-zillow",
       "requires": {
+        "@babel/plugin-proposal-optional-chaining": "^7.8.3",
```

Comment:
A lot of these local packages gained a dependency on the babel plugin? The only place this should be happening is the package.json you (correctly) changed.

Reply:
Doing Lerna right is hard